Support for Admin interface for viewing web archives

ikreymer opened the pull request:
Includes:
- Backend support to allow superadmin to view all archives
- Adding archives list on superadmin view

ikreymer committed:
support for superadmin to access all archives

SuaYoo committed:
add separate archive list view

SuaYoo commented:
Screenshots:

The instance has archives:
<img width="1068" alt="Screen Shot 2022-03-28 at 10 35 44 AM" src="https://user-images.githubusercontent.com/4672952/160455186-ee200f07-02c4-4393-a8f5-19a22eb0f82c.png">


No archives:
<img width="1033" alt="Screen Shot 2022-03-23 at 5 23 38 PM" src="https://user-images.githubusercontent.com/4672952/160450850-2cc076fc-912a-4295-9c4d-965b6dd242a8.png">

SuaYoo committed:
add spinner

SuaYoo committed:
Allow super admins to view crawl directly (#200)

ikreymer committed:
backend fixes: superadmin interface: (#201)
- support for listing all running crawls
- support for loading a single crawl by id

SuaYoo committed:
Running crawls admin view (#204)

ikreymer committed:
backend: fix list running crawls

ikreymer committed:
undo temporary changes

ikreymer committed:
reformat
